custom_components/tuya_local/devices/windmill_airconditioner.yaml

```diff
@@ -0,0 +1,53 @@
+name: Windmill Air Conditioner
+products:
+  - id: ebd7ef1c98deabe827aukn
+    name: Windmill Air Conditioner
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: wind
+              value: fan_only
+            - dps_val: cool
+              value: cool
+            - dps_val: eco
+              value: auto
+    - id: 4
+      name: mode
+      type: string
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: low
+          value: low
+        - dps_val: mid
+          value: medium
+        - dps_val: high
+          value: high
+        - dps_val: auto
+          value: auto
+    # Reported by device, seemingly unused
+    # - id: 22
+    - id: 23
+      name: current_temperature
+      type: integer
+      readonly: true
+    - id: 24
+      name: temperature
+      type: integer
+      range:
+        min: 60
+        max: 86
+      unit: F
+    # Reported by device, seemingly unused
+    # - id: 102
```

Comment:
I left these unknown params in as comments just to have them documented, and in case their use is discovered in the future. But I can take them out no problem.

Reply:
If you know the type (from logs), and they are consistently present in the data returned from the device, then it would be better to have them listed as attributes of the primary device (with names unknown_22, unknown_102) because it will help with differentiating this device from others that may be similar but not quite the same. It will also allow monitoring them from HA for anyone motivated to try to figure out what they are for (though getting access to iot.tuya.com to look up the QueryThingsDataModel result for the device is likely to be quicker than guessing by trial and error).

Reply:
Makes sense. I updated the config to add those as unknown data points.